pkg/utils/utils.go

```diff
@@ -613,6 +614,15 @@ func GetAppUidGid(pvcName string, namespace string, backup *stork_api.Applicatio
 			logrus.Errorf("%s: %v", fn, err)
 			return uid, gid, err
 		}
+
+	}
+	if uid == 0 {
```

Comment:
runAsUser: 0 is a valid value intended for pod to run as root. Not sure just checking this will ensure that we are denying for legitimate uid 0. Is there no way to figure out if it was legacy backup?

Reply:
privilege and exclusive set as root are both expects same behavior when we run our job pod in a privileged PSA setting . Hence this is been taken care.